spec/order_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'customer'
+
+RSpec.describe Order do 
+
+  let(:order) { Order.new }
+  let(:takeaway) { double('takeaway', :view => {}, :menu => { chicken: 4.99, chips: 1.50 }) }
+  
+  it 'customer can view menu' do 
+    expect(order.view).to be_instance_of Hash
```

Comment:
This test could expect that the order.view is eq to the result of takeaway.view (with some value returned), to make sure it's not overriding and returning just any hash